### DISCUSSION TITLE
Getting git-worktree to work with Lazyvim

### DISCUSSION THREAD

**Discussion by riza-khan** [LazyVim/LazyVim#4136] [Q&A]
Just switched to Lazy and can't seem to get [Git Worktree](https://github.com/polarmutex/git-worktree.nvim) plugin to work as expected. 

This is the setup:
```lua
  {
    'polarmutex/git-worktree.nvim',
    version = '^2',
    dependencies = { "nvim-lua/plenary.nvim" },
    enabled = true,
    config = function ()
      require('telescope').load_extension('git_worktree')
      vim.keymap.set( "n", "W", function ()
        require("telescope").extensions.git_worktree.git_worktrees()
      end, { desc = "Worktrees" })
  end
}
```

Throws this:

E5108: Error executing lua: /Users/[user]/.config/nvim/lua/plugins/core.lua:52: attempt to call field 'git_worktrees' (a nil value)
stack traceback:
/Users/[user]/.config/nvim/lua/plugins/core.lua:52: in function </Users/[user]/.config/nvim/lua/plugins/core.lua:51>

Any thoughts on how to get it working with Lazyvim?

**Comment by rogeriofrsouza** [comment]
Are you returning that table?
Seems like `lazy.nvim` didn't install the plugin, that's why it's 'a nil value'
Here's the docs on how to add a plugin

https://www.lazyvim.org/configuration/plugins#-adding-plugins
